types/react/experimental.d.ts

```diff
@@ -104,5 +104,5 @@ declare module '.' {
     export const SuspenseList: ExoticComponent<SuspenseListProps>;
 
     // tslint:disable-next-line ban-types
-    export function experimental_useEvent<T extends Function>(event: T): T;
+    export function experimental_useEffectEvent<T extends Function>(event: T): T;
```

Comment:
```suggestion
    /**
     * @see https://github.com/facebook/react/blob/9c54b29b44d24f8f8090da9c7ebf569747a444df/packages/react/src/ReactHooks.js#L237
     */
    export function experimental_useEffectEvent<T extends Function>(event: T): T;
```

Reply:
I don't understand what value this would add. Especially since the flow types are wrong.

Reply:
sure, no problem.  the value it would add is to point people to what this experimental thing actually does.  not important.

Reply:
The link just pointed to the implementation calling the dispatcher. That's not actually telling what it was for. We almost always want to point to documentation not implementation.